src/sidenav/components/SidenavMenu.vue

```diff
@@ -57,6 +90,10 @@ export default {
       type: Array,
       default: () => [],
     },
+    entriesMore: {
```

Comment:
You don't actually need to implement `entries-more` in this component, it's only the `SidenavGroupUI` that needs to know about the "more" bit.

Where it renders the the "more" menu it can simply use: `<SidenavMenu :entries="entriesMore" />` instead of `<SidenavMenu :entries-more="entriesMore" />`.